asreview/logging/utils.py

```diff
@@ -3,18 +3,18 @@
 
 def get_logger_class(fp):
     "Get logging class from file extension."
-    from asreview.hdf5_logging import HDF5_Logger
-    from asreview.json_logging import JSON_Logger
-    from asreview.json_logging import Dict_Logger
+    from asreview.logging.hdf5 import HDF5Logger
+    from asreview.logging.json import JSONLogger
+    from asreview.logging.dict import DictLogger
 
     if fp is None:
-        return Dict_Logger
+        return DictLogger
 
     log_ext = os.path.splitext(fp)[1]
     if log_ext in ['.h5', '.hdf5', '.he5']:
-        Logger = HDF5_Logger
+        Logger = HDF5Logger
```

Comment:
`Logger` should be lowercase (and this is not the logger defined below isn't it?).